Misc/NEWS.d/next/Build/2022-10-28-22-24-26.gh-issue-98831.IXRCRX.rst

```diff
@@ -1 +1,10 @@
-We have new tooling, in ``Tools/cases_generator``, to generate the interpreter switch from a list of opcode definitions.
+We have new tooling, in ``Tools/cases_generator``,
+to generate the interpreter switch from a list of opcode definitions.
```

Comment:
```suggestion
to generate the interpreter switch statement from a list of opcode definitions.
```

Maybe a little clearer? I was confused for a second what this was referring to until I recalled reading the issue where you described this, which mentioned this implementation detail of the core interpreter.